doc/community-build/Generate-Cortx-Build-Stack.md

```diff
@@ -11,48 +10,42 @@ To know about various CORTX components, see [CORTX Components guide](https://git
 
 ## Procedure
 
+**Note:** Run appropriate tag as per OS required i.e. CentOS 7.8 or CentOS 7.9
```

Comment:
Can we add details of OS-specific Tags. 
e.g. 
For CentOS 7.8.2003 - `docker pull ghcr.io/seagate/cortx-build:centos-7.9.2009`
For CentOS 7.9.2009 - `docker pull ghcr.io/seagate/cortx-build:centos-7.8.2003`

Reply:
done